check/lib/Util.hs

```diff
@@ -0,0 +1,59 @@
+module Util where
+
+import Data.List (isInfixOf)
+import System.Process
+
+ghci :: String -> String -> IO String
```

Comment:
If this invokes `new-repl` now, perhaps we should change the name of this function.